app/src/main/java/net/sourceforge/opencamera/sensorremote/RemoteRpcResponse.java

```diff
@@ -0,0 +1,49 @@
+package net.sourceforge.opencamera.sensorremote;
+
+import android.content.Context;
+
+import androidx.annotation.NonNull;
+
+import java.util.Properties;
+
+/**
+ * Static builder class for RPC response messages
+ */
+public class RemoteRpcResponse {
+    private String mMessage;
+
+    private RemoteRpcResponse() {
+        mMessage = "";
+    }
+
+    public static RemoteRpcResponse error(String message, Context context) {
+        Properties config = RemoteRpcConfig.getProperties(context);
+        RemoteRpcResponse result = new RemoteRpcResponse();
+        result.mMessage = config.getProperty("ERROR") +
+                "\n" +
+                message +
+                "\n" +
+                config.getProperty("CHUNK_END_DELIMITER");
+        return result;
+    }
+
+    public static RemoteRpcResponse success(String message, Context context) {
+        Properties config = RemoteRpcConfig.getProperties(context);
+        RemoteRpcResponse result = new RemoteRpcResponse();
+        result.mMessage = config.getProperty("SUCCESS") +
```

Comment:
1. Remove property access.
2. Check that compiler applies StringBuilder optimization (changes `+` concatenation to SB).